config/spryk/templates/Partials/licence.twig

```diff
@@ -1,4 +1,4 @@
 /**
```

Comment:
Could you please rename the file to `license.twig`?

Reply:
Its not a typo, but ok it should be same everywhere. fixed